### PR TITLE
Refactor camel-case method names to snake case

### PR DESCRIPTION
**Pull request by waltsims**

### CODE REVIEW

File: kwave/options.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from dataclasses import dataclass
 import numpy as np
 from kwave.utils import get_h5_literals
+from kwave.utils import get_optimal_pml_size
```
Comment (faridyagubbayli) [inline kwave/options.py:4]: How did it work before the change without this import_

File: kwave/utils/mathutils.py [hunk 2]
```diff
@@ -57,10 +57,8 @@ def fourier_shift(
         # subtract 1 in order to keep function interface compatible with matlab
         shift_dim -= 1
 
-    try:
-        N = data.shape[shift_dim]
-    except IndexError:
-        print('aaa')
+    N = data.shape[shift_dim]
```
Comment (faridyagubbayli) [inline kwave/utils/mathutils.py:60]: Good change :+1:

**Review by faridyagubbayli — APPROVED** [review]
LGTM!

File: kwave/options.py (hunk 1 above)
Comment (waltsims) [reply kwave/options.py:4]: neither do I...

**Pull request merged**